### ISSUE TITLE
Backticks in test case description cause the debugging and monitoring to fail

### ISSUE THREAD

**Issue by daelmaak** [jest-community/vscode-jest#247]
### Environment

1. `node -v`: 9.2.0
2. `npm -v`: 5.5.1
3. `npm ls react-scripts` (if you haven’t ejected): empty

4. Operating system: [fill]
Win 10

### Steps to Reproduce

![2018-02-03_21h00_37](https://user-images.githubusercontent.com/34311965/35770971-a7f995f6-0925-11e8-8b2a-dd77d6643182.gif)

1) Have a passing test with single quotes in description 
2) Then changing single quotes to backticks

### Expected Behavior

The status on the left side of the test case is green, debugging works and error messages are provided next to code in case the test is not passing.

### Actual Behavior

The status shows the test as not being run although it runs in the background. If not passing, there are no error messages next to code, only red underline on the culprit row. Debugging doesn't work in this test case.

**Comment by seanpoulter** [comment]
Thanks for reporting the issue @daelmaak.

That's a issue with `jest-editor-support`. There's an open issue for it that just needs some attention and a PR. Have a look at facebook/jest#4910.

**Comment by daelmaak** [comment]
I see you already started with the PR @seanpoulter, are you still working on that?

**Comment by seanpoulter** [comment]
Nope. I stopped with that last comment. If I was going to jump back into it, I'd want to go have a look how the AST handles tagged template literals:
```

**Comment by jkjustjoshing** [comment]
Looks like [this comment](https://github.com/facebook/jest/pull/7215#issuecomment-439109512) indicates that this might just need some version/dependency shuffling to get this working?

**Comment by connectdotz** [comment]
hi, template-literal-string support has been added to [3.0.0-preRelease](https://github.com/jest-community/vscode-jest/releases/tag/3.0.0), feel free to give it a try...
